src/common/dao/project.go

```diff
@@ -293,7 +293,7 @@ func GetRolesByLDAPGroup(projectID int64, groupDNCondition string) ([]int, error
 	}
 	o := GetOrmer()
 	sql := fmt.Sprintf(
-		`select pm.role from project_member pm 
+		`select min(pm.role) from project_member pm 
```

Comment:
Will there be sql injection issue?
And why there can be multiple ldap_group_dn?

Reply:
a user can be memberof many groups